test/aws/elbv2.go

```diff
@@ -0,0 +1,9 @@
+package aws
+
+import (
+	"github.com/aws/aws-sdk-go/service/elbv2/elbv2iface"
+)
+
+type FakeELBv2 interface {
+	elbv2iface.ELBV2API
```

Comment:
By looking at the interface AWS use, I'm pretty down to use the convention `ELBV2Repository`, wdyt?

Reply:
Agreed